modules/client/examples.md

```diff
@@ -920,7 +915,6 @@ console.log(proposal);
     address: "0x1234567890123456789012345678901234567890,
     name: "The Token",
     symbol: "TOK",
-    decimals: 18
```

Comment:
Just to confirm, aren't we returning the decimals now?

Reply:
no we are not idk why this appears on the diff on local does not appear like this